Repo {repoSshUrl = "git@github.com:jhcodeh/fail2ban.git", repoDescription = Just "Daemon to ban hosts that cause multiple authentication errors", repoCreatedAt = GithubDate {fromGithubDate = 2013-01-23 14:02:29 UTC}, repoHtmlUrl = "https://github.com/jhcodeh/fail2ban", repoSvnUrl = "https://github.com/jhcodeh/fail2ban", repoForks = 0, repoHomepage = Just "http://www.fail2ban.org", repoFork = True, repoGitUrl = "git://github.com/jhcodeh/fail2ban.git", repoPrivate = False, repoCloneUrl = "https://github.com/jhcodeh/fail2ban.git", repoSize = 2199, repoUpdatedAt = GithubDate {fromGithubDate = 2013-01-23 14:02:29 UTC}, repoWatchers = 0, repoOwner = GithubUser {githubOwnerAvatarUrl = "https://secure.gravatar.com/avatar/18633b2860eddb8acd3841d58e9077a6?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png", githubOwnerLogin = "jhcodeh", githubOwnerUrl = "https://api.github.com/users/jhcodeh", githubOwnerId = 230870, githubOwnerGravatarId = "18633b2860eddb8acd3841d58e9077a6"}, repoName = "fail2ban", repoLanguage = Just "Python", repoMasterBranch = Just "master", repoPushedAt = Just (GithubDate {fromGithubDate = 2013-01-04 20:23:21 UTC}), repoId = 7775701, repoUrl = "https://api.github.com/repos/jhcodeh/fail2ban", repoOpenIssues = 0, repoHasWiki = Just False, repoHasIssues = Just False, repoHasDownloads = Just True}